gc/stats/ConcurrentMarkPhaseStats.hpp

```diff
@@ -59,8 +57,6 @@ class MM_ConcurrentMarkPhaseStats : public MM_ConcurrentPhaseStatsBase
 
 	MM_ConcurrentMarkPhaseStats()
 		: MM_ConcurrentPhaseStatsBase(OMR_GC_CYCLE_TYPE_GLOBAL)
-		, _startTime(0)
```

Comment:
You can remove start/end times from the `clear()` function as well since the base class clear method is being called anyways